packages/neon_framework/packages/account_repository/test/account_repository_test.dart

```diff
@@ -466,46 +477,50 @@ void main() {
           emits(
             isA<_AccountStream>()
                 .having((e) => e.accounts, 'accounts', hasLength(1))
-                .having((e) => e.active, 'active', equals('b0682d652840ef50a4115cc77109bedd8c577ccc')),
+                .having((e) => e.active?.credentials, 'active', equals(credentialsList.first)),
           ),
         );
-        verify(() => storage.saveAccounts(any(that: contains(accountList.first)))).called(1);
-        verify(() => storage.saveLastAccount(any(that: equals('b0682d652840ef50a4115cc77109bedd8c577ccc')))).called(1);
+        verify(() => storage.saveAccounts(any(that: contains(credentialsList.first)))).called(1);
+        verify(() => storage.saveLastAccount(credentialsList.first.id)).called(1);
       });
     });
 
     group('logOut', () {
       setUp(() async {
-        when(() => storage.readAccounts()).thenAnswer((_) async => accountList);
+        when(() => storage.readAccounts()).thenAnswer((_) async => credentialsList);
         when(() => storage.saveLastAccount(any())).thenAnswer((_) async => {});
         when(() => storage.saveAccounts(any())).thenAnswer((_) async => {});
 
         await repository.loadAccounts();
+
+        resetMocktailState();
```

Comment:
Why is this necessary?

Reply:
Because the `await repository.loadAccounts();` will call some storage methods.
If we were to verify storage access in a test, these initial calls would throw off the call count.

This is similar to the `resetFixtures` call we have for the nextcloud package.

Reply:
Ah makes sense :+1: